inscriptions/ISic001258.xml

```diff
@@ -195,21 +200,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140746">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-
-
-                    ἀμφὶ παραστάσι ταῖσδε Σαράπιδος Ἑστίαι ἁγνὸν<lb n="2"/>βωμὸν Βαρκαῖος Καρνεάδης ἔθετο,<lb n="3"/>
-                    Εὐκρίτου υἱός, ξεῖνε, ὁ νεωκόρος ἅ θ’ ὁμόλεκτρος<lb n="4"/>Πυθιὰς ἁ κείνου καὶ θυγάτηρ Ἐρασώ.<lb n="5"/>ἀνθ’ ὧν, ὦ κραίνουσα Διὸς μεγαλαυχέας οἴκους,<lb n="6"/>θυμαρὴν βιοτᾶς ὄλβον ἔχοιεν ἀεί.
-                    <lb n="7"/>
+                    <lb n="1"/>ἀμφὶ παραστάσι ταῖσδε Σαράπιδος Ἑστίαι ἁγνὸν
+                    <lb n="2"/>βωμὸν Βαρκαῖος <name>Καρνεάδης</name> ἔθετο
+                    <lb n="3"/><name>Εὐκρίτου</name> υἱός ξεῖνε ὁ νεωκόρος ἅ θ’ ὁμόλεκτρος
+                    <lb n="4"/><name>Πυθιὰς</name> ἁ κείνου καὶ θυγάτηρ <name>Ἐρασώ</name>
+                    <lb n="5"/>ἀνθ’ ὧν ὦ κραίνουσα Διὸς μεγαλαυχέας οἴκους
+                    <lb n="6"/>θυμαρὴν βιοτᾶς ὄλβον ἔχοιεν ἀεί.
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of IG</note></app>
+                </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#GC">
+                <p>On one side of the entrance to the temple of Sarapis, Karneades of Barke, the temple-warden, son of Eukritos, dedicated this sacred pedestal to Hestia, foreigner, together with his wife Pythia and his daughter Erasos. In exchange, you reigning over Zeus' magnificent house, bring forever happiness in life.</p><!--not sure about the translation!-->
```

Comment:
pretty good I think; I'd suggest 'Near these doorposts to the temple'; and 'visitor', as a form of address from the inscription for xeine (or 'stranger'), and perhaps altar rather than pedestal; and for the last bit, 'that he (or they? can't remember my optative endings) may always have well-pleasing happiness of life' (it's horribly clunky, but I can't do much better at the moment.